debian/rules

```diff
@@ -4,6 +4,7 @@ VERSION := $(shell grep '<version>' pom.xml | head -2 | tail -1 | cut -d'>' -f2
 PACKAGE = $(shell dh_listpackages|head -n 1|cut -d '-' -f 1)
 SYSCONFDIR = "/etc"
 DESTDIR = "debian/tmp"
+CMK_REL := $(shell wget -O - "https://api.github.com/repos/apache/cloudstack-cloudmonkey/tags" 2>/dev/null | jq -r '.[0].name')
```

Comment:
noticed some rc tags, we should not keep rc tags, or make sure only release tags are picked.

```
$  wget -O - "https://api.github.com/repos/apache/cloudstack-cloudmonkey/tags" 2>/dev/null | jq -r '.[0].name'
6.4.0
$  wget -O - "https://api.github.com/repos/apache/cloudstack-cloudmonkey/tags" 2>/dev/null
[
  {
    "name": "6.4.0",
    "zipball_url": "https://api.github.com/repos/apache/cloudstack-cloudmonkey/zipball/refs/tags/6.4.0",
    "tarball_url": "https://api.github.com/repos/apache/cloudstack-cloudmonkey/tarball/refs/tags/6.4.0",
    "commit": {
      "sha": "df65df7cfe331c5af5d39743717e3d58df921a48",
      "url": "https://api.github.com/repos/apache/cloudstack-cloudmonkey/commits/df65df7cfe331c5af5d39743717e3d58df921a48"
    },
    "node_id": "MDM6UmVmMTM2NDE0Nzg6cmVmcy90YWdzLzYuNC4w"
  },
  {
    "name": "6.4.0-rc1",
    "zipball_url": "https://api.github.com/repos/apache/cloudstack-cloudmonkey/zipball/refs/tags/6.4.0-rc1",
    "tarball_url": "https://api.github.com/repos/apache/cloudstack-cloudmonkey/tarball/refs/tags/6.4.0-rc1",
    "commit": {
      "sha": "2d641ae9bfab101f7009b22ca6b2a5764224238d",
      "url": "https://api.github.com/repos/apache/cloudstack-cloudmonkey/commits/2d641ae9bfab101f7009b22ca6b2a5764224238d"
    },
    "node_id": "MDM6UmVmMTM2NDE0Nzg6cmVmcy90YWdzLzYuNC4wLXJjMQ=="
  },
...
```

Reply:
good point @sureshanaparti 
we can use `releases instead of `tags`.
it returns fewer results, but it could also include the rc releases if exists.

```
$ wget -O - "https://api.github.com/repos/apache/cloudstack-cloudmonkey/releases" 2>/dev/null | jq -r '.[].tag_name'
6.4.0
6.4.0-rc1
6.3.0
6.2.0
6.1.0
6.0.0
```

Reply:
@weizhouapache @sureshanaparti feel free to propose code changes or commit changes.

Reply:
sure @rohityadavcloud 

```suggestion
CMK_REL := $(shell wget -O - "https://api.github.com/repos/apache/cloudstack-cloudmonkey/releases" 2>/dev/null | jq -r '.[0].tag_name')
```